platform/lib/platform_web/components.ex

```diff
@@ -2391,16 +2391,54 @@ defmodule PlatformWeb.Components do
               >
                 <div class="text-left z-10">
                   <div class="h-full flex gap-1">
-                    <%= button type: "button", to: Routes.export_path(@socket, :create, @query_params),
-                      class: "rounded-full flex items-center align-center text-gray-600 hover:text-gray-600 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-offset-gray-100 focus:ring-urge-500",
-                      role: "menuitem",
-                      method: :post,
-                      "x-cloak": true,
-                      data_tooltip: "Export Incidents"
-                    do %>
-                      <Heroicons.arrow_down_tray mini class="h-5 w-5" />
-                      <span class="sr-only">Export Incidents</span>
-                    <% end %>
+                    <div x-data="{open: false}">
+                      <button
+                        x-on:click.prevent="open = !open;"
+                        type="button"
+                        class="rounded-full flex mt-px align-center items-center text-gray-600 hover:text-gray-600 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-offset-gray-100 focus:ring-urge-500"
+                        aria-expanded="true"
+                        aria-haspopup="true"
+                        data-tooltip="Export Incidents"
+                      >
+                        <span class="sr-only">Export</span>
+                        <Heroicons.arrow_down_tray mini class="h-5 w-5" />
+                      </button>
+
+                      <div
+                        x-show="open"
+                        x-on:click.outside="open = false"
+                        x-transition
+                        x-cloak
+                        class="origin-top-right absolute right-0 mt-4 w-40 rounded-md shadow-lg bg-white ring-1 ring-black ring-opacity-5 focus:outline-none"
+                        role="menu"
+                        aria-orientation="vertical"
+                        aria-labelledby="menu-button"
+                        tabindex="-1"
+                      >
+                        <div class="py-1" role="none">
+                          <%= button type: "button", to: Routes.export_path(@socket, :create_csv_export, @query_params),
+                            class: "text-gray-700 group w-full hover:bg-gray-100 flex items-center px-4 py-2 text-sm",
+                            role: "menuitem",
```

Comment:
Wanted to follow up to say that we're still learning to work with a larger team and in the future we'll be clearer and more resolute with future feature specs so that there's less avoidable back-and-forth. :)

Reply:
Re: naming, I'll follow up once we've pushed this to staging—I think it helps to name it once it's visible in the context of the rest of the UI.

Reply:
Got it! In the current version of atlos, there is an export media option in both the project page and the "search incidents" page(which I changed to the dropdown button in this PR). I will make the respective changes for the project page, but I'm wondering if the export button on the search page should be reverted to the original CSV export, or removed altogether?
Also, do we still need an entry for exporting arbitrary searches?

Reply:
>  I'm wondering if the export button on the search page should be reverted to the original CSV export, or removed altogether?

We should revert the dropdown to the original CSV export. Our logic is that because media export is only accessible to project managers and owners, exporting media from an arbitrary search across multiple projects doesn't really make sense. 

> Also, do we still need an interface for exporting arbitrary searches?

If this is in relation to the UI, we want to keep CSV export open to arbitrary searches, because arbitrary search is the primary tool users have for filtering within a project, and CSV export should be able to apply to a subset of a project. If this refers to the technical means to export arbitrary media, I'll defer to @milesmcc.

Reply:
I see, that makes sense! 
On the technical aspect, exporting ZIP for arbitrary queries is already supported, and exporting for projects would simply involve specifying the query to target the project imo.